circle.yml

```diff
@@ -86,6 +86,29 @@ commands:
         working_directory: ~/build
         command: ctest -E 'unittests|smoketests' -j4 --schedule-random --output-on-failure
 
+  collect_coverage_data:
```

Comment:
Other commands use hyphens for delimiters
```suggestion
    collect-coverage-data:
```

Reply:
I'm following CircleCI convention for command names, e.g. `store_artifacts`.

Reply:
ok, that were job names that use hyphens